draft-ietf-quic-transport.md

```diff
@@ -3700,7 +3700,7 @@ server may send multiple Initial packets.  The cryptographic key exchange could
 require multiple round trips or retransmissions of this data.
 
 The payload of an Initial packet includes a CRYPTO frame (or frames) containing
-a cryptographic handshake message, ACK frames, or both.  PADDING and
+a cryptographic handshake message, ACK frames, or both.  PADDING, PING and
```

Comment:
Not your text, but the first sentence implies an Initial packet has to contain either an ACK frame and/or a CRYPO frame, but I don't think that's true.  I'd suggest merging these sentences and say:
"The payload of an Initial packet includes a combination of one or more CRYPTO, ACK, PADDING, PING or CONNECTION_CLOSE frames."

And I'd update the paragraph below to use the same structure so they're parallel, because right now they're subtly different in not very interesting ways.